src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -14,327 +12,23 @@ namespace System.Xaml.Permissions
-                {
-                    throw new ArgumentException(SR.Get(SRID.CollectionCannotContainNulls, "allowedAccess"));
-                }
-                accessList.Add(accessLevel);
-            }
-            Init(false, accessList);
-        }
-
+        public XamlLoadPermission(PermissionState state) { }
+        public XamlLoadPermission(XamlAccessLevel allowedAccess) { }
+        public XamlLoadPermission(IEnumerable<XamlAccessLevel> allowedAccess) { }
 #if NETCOREAPP3_0
-
         [ComVisible(false)]
-        public override bool Equals(object obj)
-        {
-            IPermission perm = obj as IPermission;
-            if (obj != null && perm == null)
-            {
-                return false;
-            }
-
-            try
-            {
-                if (!IsSubsetOf(perm))
-                {
-                    return false;
-                }
-
-                if (perm != null && !perm.IsSubsetOf(this))
-                {
-                    return false;
-                }
-            }
-            catch (ArgumentException)
-            {
-                // Any argument exception implies inequality
-                // Note that we require a try/catch here because we have to deal with
-                // custom permissions that may throw exceptions indiscriminately
-                return false;
-            }
-
-            return true;
-        }
-
+        public override bool Equals(object obj) { return false; }
         [ComVisible(false)]
-        public override int GetHashCode()
-        {
-            // This implementation is only to silence a compiler warning
-            return base.GetHashCode();
-        }
-
+        public override int GetHashCode() { return base.GetHashCode(); }
 #endif 
-
-        // copy ctor. We can reuse the list of the existing instance, because it is a
-        // ReadOnlyCollection over a privately created array, hence is never mutated,
-        // even if the other instance is mutated via FromXml().
-        private XamlLoadPermission(XamlLoadPermission other)
-        {
-            _isUnrestricted = other._isUnrestricted;
-            AllowedAccess = other.AllowedAccess;
-        }
-
-        private void Init(bool isUnrestricted, IList<XamlAccessLevel> allowedAccess)
-        {
-            _isUnrestricted = isUnrestricted;
-            if (allowedAccess == null)
-            {
-                if (s_emptyAccessLevel == null)
-                {
-                    s_emptyAccessLevel = new ReadOnlyCollection<XamlAccessLevel>(Array.Empty<XamlAccessLevel>());
-                }
-                AllowedAccess = s_emptyAccessLevel;
-            }
-            else
-            {
-                Debug.Assert(!isUnrestricted);
-                AllowedAccess = new ReadOnlyCollection<XamlAccessLevel>(allowedAccess);
-            }
-        }
-
         public IList<XamlAccessLevel> AllowedAccess { get; private set; } // ReadOnlyCollection
```

Comment:
Remove comment

Reply:
You may still want to back this with an actual empty readonly list so that you provide an instance as before instead of null.